client/components/pages/WelcomePage/WelcomePage.js

```diff
@@ -1,31 +1,39 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { push } from 'connected-react-router';
 import R from 'ramda';
+import { getNoaWelcoming, msg } from '/Users/macbookair/Documents/goodforest/goodforest/client/api/noaExample.js';
```

Comment:
imports are made relative to the location of the file. for example,
import SettingsMenu from '_organisms/SettingsMenu';